solar_system.rb

```diff
@@ -0,0 +1,35 @@
+
+class SolarSystem
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+  end
+
+  attr_reader :star_name, :planets
```

Comment:
It's convention to put the `attr_reader` and `attr_accessor` etc above `initialize` and below the class name